src/sidebar/helpers/thread-sorters.ts

```diff
@@ -112,4 +98,4 @@ export const sorters = {
     const bPos = bLocation.position ?? Number.MAX_SAFE_INTEGER;
     return Math.sign(aPos - bPos);
   },
-};
+} satisfies Record<string, SortFunction>;
```

Comment:
I think we ran into some issue the first time we tried to use `satisfies`. It looks like this has been resolved by updates since then.

Reply:
This is actually the first time I use `satisfies` for something useful 😅 

My first approach was to just define the type as `Record<string, SortFunction>`, but there's another place where we do `keyof typeof sorters`, which was just resolving to `string` in that case.

Using `satisfies` allows for new keys to be added, while properly resolves the subset of actual keys when doing the aforementioned `keyof typeof sorters`.